### ISSUE TITLE
Allow user to set limits for alarm

### ISSUE THREAD

**Issue by sabinammm** [cadus/remo2hbo_oktopus_frontend#4]
Threshold values that define the normal clinical range of a vital sign can vary according to age, sex, or disease. It should therefore be possible to set the lower and upper threshold values at which an alarm signal will be set off for each patient individually.

**Comment by danhauck** [comment]
So to clarify a bit. The user should be able to choose a window (upper and lower threshold) for Temp °C, HR / min, spO2 %, Sys / Dia mmHG. If current value leaves the window an alarm shall visualized. Is this correct? Should also an acustical alarm occure?

**Comment by sabinammm** [comment]
Yes, exactly to both. An acoustical alarm would be preferential, if a visual alarm is more easy to implement initially this could be an incremental first step.
